index.js

```diff
@@ -118,10 +116,13 @@ sassLint.lintText = function (file, options, configPath) {
       severity: 2
     }];
   }
-
-  if (ast.content && ast.content.length > 0) {
+  if (ast.tree && ast.tree.content && ast.tree.content.length > 0) {
     rules.forEach(function (rule) {
-      detects = rule.rule.detect(ast, rule);
+      detects = rule.rule.detect(ast.tree, rule);
```

Comment:
Do not complect the fix code with the lint code. These two things should live separately.